flink-runtime/src/main/java/org/apache/flink/runtime/scheduler/SlotSharingExecutionSlotAllocatorFactory.java

```diff
@@ -84,4 +96,25 @@ public ExecutionSlotAllocator createInstance(final ExecutionSlotAllocationContex
                 allocationTimeout,
                 context::getResourceProfile);
     }
+
+    private static SlotSharingStrategy.Factory getSlotSharingStrategyFactory(
+            @Nullable Configuration configuration) {
```

Comment:
I checked the parameter passing link. There will be no null values of `configuration`, I will remove this null-configuration judgment.